---
layout: single
classes: wide
author_profile: false
permalink: /publications/
title: "Publications"
toc: true
toc-title: "Contents"
---

Follow my research on [Google Scholar](https://scholar.google.com/citations?user=xxVfsMIAAAAJ&hl=en) and [ResearchGate](https://www.researchgate.net/profile/Clifton-Mckee).

## Peer-reviewed articles

<sup>†</sup>Equal contribution, <sup>‡</sup>Student advisee

**2024**

**McKee CD**, Peel AJ, Hayman DTS, Suu-Ire R, Ntiamoa-Baidu Y, Cunningham AA, Wood JLN, Webb CT, Kosoy MY. Ectoparasite and bacterial population genetics and community structure indicate extent of bat movement across an island chain. [_In press at Parasitology_](https://doi.org/10.1017/S0031182024000660). [**PDF**](http://clifmckee.github.io/files/2024_McKee_Parasitology.pdf)

Cortes-Azuero O, Lefrancq N, Nikolay B, **McKee C**, Cappelle J, Hul V, Ou TP, Hoem T, Lemey P, Rahman MZ, Islam A, Gurley ES, Duong V, Salje H. The genetic diversity of Nipah virus across spatial scales. [The Journal of Infectious Diseases. 2024; jiae221](https://doi.org/10.1093/infdis/jiae221). [**PDF**](http://clifmckee.github.io/files/2024_Cortes-Azuero_JID.pdf)

Jung S-m, Loo SL, Howerton E, Contamin L, Smith CP, Carcelén EC, Yan K, Bents SJ, Levander J, Espino J, Lemaitre JC, ..., **McKee CD**, ..., Lessler J. Potential impact of annual vaccination with reformulated COVID-19 vaccines: lessons from the US COVID-19 scenario modeling hub. [PLOS Medicine. 2024; 21(4): e1004387](https://doi.org/10.1371/journal.pmed.1004387). [**PDF**](http://clifmckee.github.io/files/2024_Jung_PLoS_Med.pdf)

Lemaitre JC, Loo SL, Kaminsky J, Lee EC, **McKee C**, Smith C, Jung S-m, Sato K, Carcelen E, Hill A, Lessler J, Truelove S. *flepiMoP*: the evolution of a flexible infectious disease modeling pipeline during the COVID-19 pandemic. [Epidemics. 2024; 47: 100753](https://doi.org/10.1016/j.epidem.2024.100753). [**PDF**](http://clifmckee.github.io/files/2024_Lemaitre_Epidemics.pdf)

**2023**

Howerton E, Contamin L, Mullany LC, Qin MM, Reich NG, Bents SJ, Borchering RK, Jung SM, Loo SL, Smith CP, Levander J, ..., **McKee C**, ..., Viboud C, Lessler J. Evaluation of the US COVID-19 Scenario Modeling Hub for informing pandemic response under uncertainty. [Nature Communications. 2023; 14: 7260](https://doi.org/10.1038/s41467-023-42680-x). [**PDF**](http://clifmckee.github.io/files/2023_Howerton_Nat_Commun.pdf)

Fagre AC, Islam A, Reeves WK, Kading RC, Plowright RK, Gurley ES, **McKee CD**. *Bartonella* infection in fruit bats and bat flies, Bangladesh. [Microbial Ecology. 2023; 86: 2910–2922](https://doi.org/10.1007/s00248-023-02293-9). [**PDF**](http://clifmckee.github.io/files/2023_Fagre_Microb_Ecol.pdf)

Szentiványi T, **McKee C**, Jones G, Foster JT. Trends in bacterial pathogens of bats: global distribution and knowledge gaps. [Transboundary and Emerging Diseases. 2023; 9285855](https://doi.org/10.1155/2023/9285855). [**PDF**](http://clifmckee.github.io/files/2023_Szentivanyi_TBED.pdf)

**2022**

Seidlova V, Straková P, Kejíková R, Nemcova M, Bartonička T, Salát J, Dufková L, Šikutová S, Mendel J, **McKee C**, Zukal J, Pikula J, Rudolf I. Detection of *Leptospira* species in bat cadavers, Czech and Slovak Republics. [Emerging Microbes & Infections. 2022; 11(1): 2211--2213](https://doi.org/10.1080/22221751.2022.2117095). [**PDF**](http://clifmckee.github.io/files/2022_Seidlova_Emerg_Microbes_Infect.pdf)

Kejíková R, **McKee C**, Straková P, Šikutová S, Mendel J, Rudolf I. First detection of *Bartonella* spp. in bat bugs *Cimex pipistrelli* (Hemiptera: Cimicidae), Central Europe. [Parasitology Research. 2022; 121: 3341--3345](https://doi.org/10.1007/s00436-022-07668-4). [**PDF**](http://clifmckee.github.io/files/2022_Kejikova_Parasitology_Res.pdf)

Goodrich I, **McKee C**, Margos G, Kosoy M. Molecular characterization of a novel relapsing fever *Borrelia* species from the desert cottontail (*Sylvilagus audubonii*) in New Mexico, USA. [Journal of Wildlife Diseases. 2022; 58(3): 646--651](https://doi.org/10.7589/JWD-D-21-00148). [**PDF**](http://clifmckee.github.io/files/2022_Goodrich_JWD.pdf)

**McKee CD**<sup>†</sup>, Islam A<sup>†</sup>, Rahman MZ, Khan SU, Rahman M, Satter SM, Islam A, Yinda CK, Epstein JH, Daszak P, Munster VJ, Hudson PJ, Plowright RK, Luby SP, Gurley ES. Nipah virus detection at bat roosts after spillover events, Bangladesh, 2012–2019. [Emerging Infectious Diseases. 2022; 28(7): 1384--1392](https://doi.org/10.3201/eid2807.212614). [**PDF**](http://clifmckee.github.io/files/2022_McKee_EID.pdf)

Rice BL<sup>†</sup>, Lessler J<sup>†</sup>, **McKee C**<sup>†</sup>, Metcalf CJE<sup>†</sup>. Why do some coronaviruses become pandemic threats when others do not? [PLOS Biology. 2022; 20(5): e3001652](https://doi.org/10.1371/journal.pbio.3001652). [**PDF**](http://clifmckee.github.io/files/2022_Rice_PLoS_Biol.pdf)

Ruiz-Aravena M<sup>†</sup>, **McKee C**<sup>†</sup>, Gamble A, Lunn T, Morris A, Snedden CE, Yinda CK, Port JR, Buchholz DW, Yeo YY, Faust C, ..., Munster VJ, Plowright RK. Ecology, evolution and spillover of coronaviruses from bats. [Nature Reviews Microbiology. 2022; 20: 299--314](https://doi.org/10.1038/s41579-021-00652-2). [**PDF**](http://clifmckee.github.io/files/2022_Ruiz-Aravena_Nat_Rev_Microbiol.pdf)

Redd AD, Peetluk LS, Jarrett BA, Hanrahan C, Schwartz S, Rao A, Jaffe AE, Peer AD, Jones CB, Lutz CS, **McKee CD**, ..., Grabowski MK, Gurley ES, the Novel Coronavirus Research Compendium Team. Curating the evidence about COVID-19 for frontline public health and clinical care: the Novel Coronavirus Research Compendium. [Public Health Reports. 2022; 137(2): 197--202](https://doi.org/10.1177/00333549211058732). [**PDF**](http://clifmckee.github.io/files/2022_Redd_PHR.pdf)

**2021**

Islam A, **McKee C**, Ghosh PK, Abedin J, Epstein JH, Daszak P, Luby SP, Khan SU, Gurley ES. Seasonality of date palm sap feeding behavior by bats in Bangladesh. [EcoHealth. 2021; 18: 359--371](https://doi.org/10.1007/s10393-021-01561-9). [**PDF**](http://clifmckee.github.io/files/2021_Islam_EcoHealth.pdf)

Zorrilla VO, Lozano ME, Espada LJ, Kosoy M, **McKee C**, Valdivia HO, Arevalo H, Troyes M, Stoops CA, Fisher ML, Vásquez GM. Comparison of sand fly trapping approaches for vector surveillance of *Leishmania* and *Bartonella* species in ecologically distinct, endemic regions of Peru. [PLOS Neglected Tropical Diseases. 2021; 15(7): e0009517](https://doi.org/10.1371/journal.pntd.0009517). [**PDF**](http://clifmckee.github.io/files/2021_Zorrilla_PLoS_NTD.pdf)

**McKee CD**, Islam A, Luby SP, Salje H, Hudson PJ, Plowright RK, Gurley ES. The ecology of Nipah virus in Bangladesh: a nexus of land-use change and opportunistic feeding behavior in bats. [Viruses. 2021; 13(2): 169](https://doi.org/10.3390/v13020169). [**PDF**](http://clifmckee.github.io/files/2021_McKee_Viruses.pdf)

**McKee C**, Bai Y, Webb C, Kosoy M. Bats are key hosts in the radiation of mammal-associated *Bartonella* bacteria. [Infection, Genetics and Evolution. 2021; 89: 104719](https://doi.org/10.1016/j.meegid.2021.104719). [**PDF**](http://clifmckee.github.io/files/2021_McKee_IGE.pdf)

**2020**

Goodrich I, **McKee C**, Kosoy M. *Trypanosoma* (*Herpetosoma*) diversity in rodents and lagomorphs of New Mexico with a focus on epizootological aspects of infection in Southern Plains woodrats (*Neotoma micropus*). [PLOS ONE. 2020; 15(12): e0244803](https://doi.org/10.1371/journal.pone.0244803). [**PDF**](http://clifmckee.github.io/files/2020_Goodrich_PLoS_ONE.pdf)

Rudolf I, Blažejová H, Mendel J, Straková P, Šebesta O, Rettich F, Čabanová V, Miterpáková M, Betášová L, Peško J, Barbušinová E, **McKee C**, Osikowicz L, Šikutová S, Hubálek Z, Kosoy M. *Bartonella* species in medically important mosquitoes, Central Europe. [Parasitology Research. 2020; 119(8): 2713--2717](https://doi.org/10.1007/s00436-020-06732-1). [**PDF**](http://clifmckee.github.io/files/2020_Rudolf_Parasitology_Res.pdf)

Goodrich I, **McKee C**, Kosoy M. Longitudinal study of bacterial infectious agents in a community of small mammals in New Mexico. [Vector-Borne and Zoonotic Diseases. 2020; 20(7): 496--508](https://doi.org/10.1089/vbz.2019.2550). [**PDF**](http://clifmckee.github.io/files/2020_Goodrich_VBZD.pdf)

**2019**

**McKee CD**, Krawczyk AI, Sándor AD, Görföl T, Földvári M, Földvári G, Dekeukeleire D, Haarsma A-J, Kosoy MY, Webb CT, Sprong H. Host phylogeny, geographic overlap, and roost sharing shape parasite communities in European bats. [Frontiers in Ecology and Evolution. 2019; 7: 69](https://doi.org/10.3389/fevo.2019.00069). [**PDF**](http://clifmckee.github.io/files/2019_McKee_Front_Ecol_Evol.pdf)

**2018**

Bai Y, Osinubi MOV, Osikowicz L, **McKee C**, Vora NM, Rizzo MR, Recuenco S, Davis L, Niezgoda M, Ehimiyein AM, Kia GSN, Oyemakinde A, Adeniyi OS, Gbadegesin YH, Saliman OA, Ogunniyi A, Ogunkoya AB, Kosoy MY, Idanre Bat Festival Investigation Team. Human exposure to novel *Bartonella* species from contact with fruit bats. [Emerging Infectious Diseases. 2018; 24(12): 2317--23](https://doi.org/10.3201/eid2412.181204). [**PDF**](http://clifmckee.github.io/files/2018_Bai_EID.pdf)

Kellner A, Carver S, Scorza V, **McKee CD**, Lappin M, Crooks KR, VandeWoude S, Antolin MF. Transmission pathways and spillover of an erythrocytic bacterial pathogen from domestic cats to wild felids. [Ecology and Evolution. 2018; 8(19): 9779--9792](https://doi.org/10.1002/ece3.4451). [**PDF**](http://clifmckee.github.io/files/2018_Kellner_Ecol_Evol.pdf)

**McKee CD**, Osikowicz LM, Schwedhelm TR, Maes SE, Enscore RE, Gage KL, Kosoy MY. Acquisition of *Bartonella elizabethae* by experimentally exposed oriental rat fleas (*Xenopsylla cheopis*; Siphonaptera, Pulicidae) and excretion of *Bartonella* DNA in flea feces. [Journal of Medical Entomology. 2018; 55(5): 1292--1298](https://doi.org/10.1093/jme/tjy085). [**PDF**](http://clifmckee.github.io/files/2018b_McKee_JME.pdf)

Kosoy M<sup>†</sup>, **McKee C**<sup>†</sup>, Albayrak L, Fofanov Y. Genotyping of *Bartonella* bacteria and their animal hosts: current status and perspectives. [Parasitology. 2018; 145(5): 543--562](https://doi.org/10.1017/S0031182017001263). [**PDF**](http://clifmckee.github.io/files/2018_Kosoy_Parasitology.pdf)

Gorsich EE, **McKee CD**, Grear DA, Miller RS, Portacci K, Lindström T, Webb CT. Model-guided suggestions for targeted surveillance based on cattle shipments in the U.S. [Preventive Veterinary Medicine. 2018; 150: 52--59](https://doi.org/10.1016/j.prevetmed.2017.12.004). [**PDF**](http://clifmckee.github.io/files/2018_Gorsich_PVM.pdf)

**McKee CD**, Osikowicz LM, Schwedhelm TR, Bai Y, Castle KT, Kosoy MY. Survey of parasitic bacteria in bat bugs, Colorado. [Journal of Medical Entomology. 2018; 55(1): 237--241](https://doi.org/10.1093/jme/tjx155). [**PDF**](http://clifmckee.github.io/files/2018a_McKee_JME.pdf)

**2017**

**McKee CD**, Kosoy MY, Bai Y, Osikowicz LM, Franka R, Gilbert AT, Boonmar S, Rupprecht CE, Peruski LF. Diversity and phylogenetic relationships among *Bartonella* strains from Thai bats. [PLOS ONE. 2017; 12(7): e0181696](https://doi.org/10.1371/journal.pone.0181696). [**PDF**](http://clifmckee.github.io/files/2017_McKee_PLoS_ONE.pdf)

Urushadze L, Bai Y, Osikowicz L, **McKee C**, Sidamonidze K, Putkaradze D, Imnadze P, Kandaurov A, Kuzmin I, Kosoy M. Prevalence, diversity, and host associations of *Bartonella* strains in bats from Georgia (Caucasus). [PLOS Neglected Tropical Diseases. 2017; 11(4): e0005428](https://doi.org/10.1371/journal.pntd.0005428). [**PDF**](http://clifmckee.github.io/files/2017_Urushadze_PLoS_NTD.pdf)

Bai Y, Urushadze L, Osikowicz L, **McKee C**, Kuzmin I, Kandaurov A, Babuadze G, Natradze I, Imnadze P, Kosoy M. Molecular survey of bacterial zoonotic agents in bats from the country of Georgia (Caucasus). [PLOS ONE. 2017; 12(1): e0171175](https://doi.org/10.1371/journal.pone.0171175). [**PDF**](http://clifmckee.github.io/files/2017_Bai_PLoS_ONE.pdf)

**2016**

**McKee CD**, Hayman DTS, Kosoy MY, Webb CT. Phylogenetic and geographic patterns of bartonella host shifts among bat species. [Infection, Genetics and Evolution. 2016; 44: 382--394](https://doi.org/10.1016/j.meegid.2016.07.033). [**PDF**](http://clifmckee.github.io/files/2016_McKee_IGE.pdf)

**2015**

Zinzow-Kramer WM, Horton BM, **McKee CD**, Michaud JM, Tharp GK, Thomas JW, Tuttle EM, Yi S, Maney DL. Genes located in a chromosomal inversion are correlated with territorial song in white-throated sparrows. [Genes, Brain and Behavior. 2015; 14(8): 641--654](https://doi.org/10.1111/gbb.12252). [**PDF**](http://clifmckee.github.io/files/2015_ZinzowKramer_GBB.pdf)

Bai Y, Hayman DTS, **McKee CD**, Kosoy MY. Classification of *Bartonella* strains associated with straw-colored fruit bats (*Eidolon helvum*) across Africa using a multi-locus sequence typing platform. [PLOS Neglected Tropical Diseases. 2015; 9(1): e0003478](https://doi.org/10.1371/journal.pntd.0003478). [**PDF**](http://clifmckee.github.io/files/2015_Bai_PLoS_NTD.pdf)

## Preprints

Stevens T, Zimmerman R, Albery G, Becker DJ, Kading R, Keiser CN, Khandelwal S, Kramer-Schadt S, Krut-Landau R, **McKee C**, Montecino-Latorre D, ..., Carlson CJ. A minimum data standard for wildlife disease studies. [EcoEvoRxiv. 2024](https://doi.org/10.32942/x2tw4j). [**PDF**](http://clifmckee.github.io/files/2024_Stevens_EcoEvoRxiv.pdf)

Jackson J<sup>‡</sup>, Shanta IS, **McKee C**, Luby SP, Haider N, Sharker Y, Plowright R, Hudson P, Gurley E. Identifying weather patterns affecting household date palm sap consumption in Bangladesh, 2013-2016. [medRxiv. 2024](https://doi.org/10.1101/2024.05.06.24306951). [**PDF**](http://clifmckee.github.io/files/2024_Jackson_medRxiv.pdf)

**McKee CD**, Yu EX, Garcia A, Jackson J, Koyuncu A, Rose S, Azman AS, Lobner K, Sacks E, Van Kerkhove MD, Gurley ES. Superspreading of SARS-CoV-2: a systematic review and meta-analysis of event attack rates and individual transmission patterns. [medRxiv. 2024](https://doi.org/10.1101/2024.01.25.24301669). [**PDF**](http://clifmckee.github.io/files/2024_McKee_medRxiv.pdf)

Mathis SM, Webber AE, Basu A, Drake JM, White LA, Murray EL, Sun M, Leon TM, Hu AJ, Shemetov D, Brooks LC, ..., **McKee CD**, ..., Borchering RK. Evaluation of FluSight influenza forecasting in the 2021-22 and 2022-23 seasons with a new target laboratory-confirmed influenza hospitalizations. [medRxiv. 2023](https://doi.org/10.1101/2023.12.08.23299726). [**PDF**](http://clifmckee.github.io/files/2023_Mathis_medRxiv.pdf)

**McKee CD**, Webb CT, Kosoy MY, Bai Y, Osikowicz LM, Suu-Ire R, Ntiamoa-Baidu Y, Cunningham AA, Wood JL, Hayman DT. Manipulating vector transmission reveals local processes in bacterial communities of bats. [bioRxiv. 2021](https://doi.org/10.1101/2021.03.03.433743). [**PDF**](http://clifmckee.github.io/files/2021_McKee_bioRxiv.pdf)

## Other contributions

The Flexible Epidemic Modeling Pipeline (*flepiMoP*): a software suite for simulating a wide range of compartmental models of infectious disease transmission. Available at: [https://www.flepimop.org/](https://www.flepimop.org/).

2019 Novel Coronavirus Research Compendium (NCRC). 2021. Available at: [https://ncrc.jhsph.edu/](https://ncrc.jhsph.edu/).

Owers K, McKee C, Hallman C, Portacci L, Miller R, Lindström T, Webb C. 2015. USAMM R Shiny Visualization. Available at: [https://usamm-gen-net.shinyapps.io/usamm-gen-net/](https://usamm-gen-net.shinyapps.io/usamm-gen-net/).
